### ISSUE TITLE
Preferences' panes not selectable (well) with VoiceOver

### ISSUE THREAD

**Issue by dusek** [ViennaRSS/vienna-rss#260]
VoiceOver user cannot switch between Preferences panes ("General", "Appearance", "Syncing", "Advanced") in a standard way (with VO-Space).

Repro:
1. Open Preferences (Cmd-,)
2. Launch VoiceOver (Cmd-F5)
3. Interact with the toolbar (VO-Shift-arrow down)
4. Move VoiceOver cursor to the unselected "Appearance" button (VO-arrow right)
5. Press "Appearance" button (VO-Space)

Expected Results:
"Appearance" pane would get displayed in Preferences.

Actual Results:
"General" pane stays displayed in Preferences. "Appearance" button visually appears as going through process of being pressed, but this has no effect in the end.

Versions:
OS X 10.9.2 (13C64)
Vienna Version Master :a87dd53: (3.0.0 Beta 20+)

Workaround:
Instead of step 5., do: VO-Cmd-F5 (move mouse pointer to VoiceOver cursor) then VO-Shift-Space (click).
